buzzmobile/drive/horn/src/horn_node.cpp

```diff
@@ -0,0 +1,42 @@
+#include <ros/ros.h>
+#include <std_msgs/Bool.h>
+#include <buzzmobile/CarPose.h>
+
+std_msgs::Bool on;
+std_msgs::Bool off;
+ros::Publisher car_horn_pub;
+
+/*void honkLongHandler(const std_msgs::Bool::ConstPtr&) {
+//  std::cout << "In honkLong" << std::endl;
+  car_horn_pub.publish(on);
+  ros::Duration(0.5).sleep(); // sleep for half a second
+  car_horn_pub.publish(off);
+//  std::cout << "Exiting honkLong" << std::endl;
+}*/
```

Comment:
Also fine with these commented out things because we have open issues for handling these in the future #41 #66